azure/citus-bot.sh

```diff
@@ -40,8 +40,19 @@ trap cleanup EXIT
 
 rg=$1
 export RESOURCE_GROUP_NAME=${rg}
-./create-cluster.sh
 
+if [ "$rg" == "citusbot_valgrind_test_resource_group" ]; then
```

Comment:
We kind of want to do the cleanup if it fails while creating the cluster I think, but we need the VALGRIND_TEST variable set before running ./create-cluster.sh

We can possibly do the following:

```bash
if [ "$rg" == "citusbot_valgrind_test_resource_group" ]; then
    # If running valgrind tests, export VALGRIND_TEST to be 1 to ensure
    # only coordinator instance is created in create-cluster script
    export VALGRIND_TEST=1
fi

./create-cluster.sh
if <VALGRIND_TEST variable = 1> ; then
    # If running valgrind tests, do not run cleanup function
    # This is because, as valgrind tests requires too much time to run,
    # we start valgrind tests via nohup in ci. Hence ssh session
    # will immediately be closed just after the fabric command is run
    trap - EXIT
fi
 
```

Reply:
makes a lot sense, thanks